Makefile

```diff
@@ -99,27 +99,40 @@ release: \
 
 release-victoria-metrics: \
 	release-victoria-metrics-amd64 \
+	release-victoria-metrics-linux-amd64 \
+	release-victoria-metrics-darwin-amd64 \
```

Comment:
It is better from readability PoV to move this target closer to `release-victoria-metrics-darwin-arm64`